src/openms/source/ANALYSIS/OPENSWATH/SwathMapMassCorrection.cpp

```diff
@@ -238,18 +238,22 @@ namespace OpenMS
       {
         if (ms1_im_)
         {
-          sp_ms1 = OpenSwathScoring().fetchSpectrumSwath(ms1_maps, bestRT, 1, 0, 0);
+          std::vector<OpenSwath::SpectrumPtr> fetchSpectrumArr = OpenSwathScoring().fetchSpectrumSwath(ms1_maps, bestRT, 1, 0, 0);
```

Comment:
this ` .. 1, 0, 0` looks like its easy to use wrong. Again, a DriftTime range as a single argument would be nice.